lib/workers/repository/process/lookup/index.spec.ts

```diff
@@ -783,6 +783,33 @@ describe(getName(), () => {
       expect(res.updates).toHaveLength(1);
       expect(res.updates[0].newValue).toEqual('3.0.1');
     });
+    it('should update pinned versions if updatePinnedDependencies=true', async () => {
+      config.currentValue = '0.0.34';
+      config.updatePinnedDependencies = true;
+      config.depName = '@types/helmet';
+      config.datasource = datasourceNpmId;
+      httpMock
+        .scope('https://registry.npmjs.org')
+        .get('/@types%2Fhelmet')
+        .reply(200, helmetJson);
+      const res = await lookup.lookupUpdates(config);
+      // expect(res.updates).toMatchSnapshot(); //TODO
+      expect(res.updates).toHaveLength(1);
+      expect(res.updates[0].newValue).toEqual('0.0.35');
+    });
+    it('should not update pinned versions if updatePinnedDependencies=false', async () => {
+      config.currentValue = '0.0.34';
+      config.updatePinnedDependencies = false;
+      config.depName = '@types/helmet';
+      config.datasource = datasourceNpmId;
+      httpMock
+        .scope('https://registry.npmjs.org')
+        .get('/@types%2Fhelmet')
+        .reply(200, helmetJson);
+      const res = await lookup.lookupUpdates(config);
+      // expect(res.updates).toMatchSnapshot(); //TODO
+      expect(res.updates).toHaveLength(0);
+    });
```

Comment:
```suggestion
    });

```

Newlines for readability.

Reply:
Done, but
1) Shouldn't all the other places in this file be fixed too(there is only a couple of newlines in this whole file)?
2) Shouldn't it be added to linter?

Reply:
1. For now we're just trying to improve the whitespace for anything which is new/changed, instead of doing bulk changes or asking PR authors to make unrelated changes
2. I'm not aware of such an automated rule existing

Reply:
2. There is a eslint plugin https://github.com/dangreenisrael/eslint-plugin-jest-formatting

I already thought about it, but not yet hat time to test